cmd/kube-scheduler/app/server_test.go

```diff
@@ -287,3 +295,44 @@ profiles:
 		})
 	}
 }
+
+func TestNewHealthzHandler(t *testing.T) {
+	// Setup any healthz checks we will want to use.
+	var checks []healthz.HealthChecker
+
+	//Test1: EnableProfiling=true
+	c := &kubeschedulerconfig.KubeSchedulerConfiguration{
+		DebuggingConfiguration: componentbaseconfig.DebuggingConfiguration{
+			EnableProfiling:           true,
+			EnableContentionProfiling: true,
```

Comment:
Looks like you mostly want to test the enabling of profiling? if that is the case the name of the testcase and pr subject should be updated accordingly.

Reply:
thanks，i have updated.

Reply:
Make it a testtable